app/you/preferences.tsx

```diff
@@ -108,9 +108,10 @@ export default function Preferences() {
         <SingleClickTouchableOpacity
           delay={500}
           activeOpacity={1}
-          onPress={() => setSwitchOn(!isSwitchOn)}>
+          onPress={() => setSwitchOn(!isSwitchOn)}
+          disabled={preferences.mode === 'system'}>
           <CustomTable title={t('Dark mode')}>
-            <CustomSwitch disabled={preferences.mode === 'system'} value={isSwitchOn} />
```

Comment:
the property disabled must stay because based on it switch changes its colors

Reply:
done